community.json

```diff
@@ -256,7 +256,7 @@
     },
     "distbin": {
         "branch": "master",
-        "level": 5,
+        "level": 3,
```

Comment:
For distbin, all test indicate at least a level 5.
https://dash.yunohost.org/appci/app/distbin

But in reality, it's level 7 but the community.json has not been updated in the CI, so the CI still use the old url: https://github.com/yalh76/distbin_ynh

Reply:
@Psycojoker, is it fixed in YunoRunner and not up to date on the CI ? Or is it still something to fix ?

Anyway, the community list is indeed modified. https://github.com/YunoHost/apps/commit/1c2ebe65f75427e3c969e5bc70e24796d85541ae

Reply:
> @Psycojoker, is it fixed in YunoRunner and not up to date on the CI ?

Yes it's fixed in YunoRunner, that's probably an outdated CI.

Reply:
https://github.com/YunoHost/yunorunner/commit/a9cdea0ee2119eff768c911b0cb47b856c29f4e7

Reply:
Yes, https://github.com/YunoHost/apps/blob/master/community.json is fine.

https://yunohost.org/community.json is at least 5 days late (I changed the maintainer for distbin https://github.com/YunoHost-Apps/distbin_ynh/commit/511e92911496f4a1fc10a308c190492c0fcbc527#diff-4b1eb3dc48c4e16d49db5b42298fe654
![capture](https://user-images.githubusercontent.com/30271971/52657915-07daeb00-2efa-11e9-8d49-12449f9b5214.PNG)
, but this list don't show the changes:
![capture](https://user-images.githubusercontent.com/30271971/52658003-4375b500-2efa-11e9-8568-d54c424f2ef7.PNG)

And YunoRunner seems to work on a 13 days old communty.json..

I say that but I really don't know how long this lists are recalculated/updated ^^

Reply:
BTW: a big thank you for the time you spend on Yunohost and sorry to waste a bit of your time ^^ I'm learning.

Reply:
I don't think you're wasting anybody's time, on the contary, you found a bug which otherwise would have gone unnoticed for a few more weeks :stuck_out_tongue_winking_eye:

Reply:
About YunoRunner, I'm finishing my work on Dokuwiki and I'll take care of that.

Reply:
Done https://github.com/YunoHost-Apps/yunorunner_ynh_core/commit/07a208cda2eb8b6192027e00219b26a85556edc2
I'm going to upgrade each CI